yarn.lock

```diff
@@ -3060,6 +3060,139 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@backstage/cli@npm:^0.25.3-next.1":
```

Comment:
i see there's a couple of new backstage library dep versions here, does the new internal package need to "unify" to some certain set of ranges that match the current release the rest of the repo is on?